src/library/scala/concurrent/Promise.scala

```diff
@@ -12,7 +12,7 @@
 
 package scala.concurrent
 
-import scala.util.{ Try, Success, Failure }
+import scala.Try.{Failure, Success}
```

Comment:
There is a whole culture war still to be fought over the order of `Success` and `Failure`. A namespace that allowed wildcard import would be a no-brainer. I'm not convinced that moving them into the companion pays for itself.

Reply:
Yes, should be `import scala.Try.{Success, Failure}` for minimal code changing.